config/routes.rb

```diff
@@ -75,6 +75,10 @@
   namespace :admin do
     get 'dashboard', to: 'dashboard#show'
 
+    resources :depositors_search, only: %i[new] do
+      collection { post :index }
+    end
```

Comment:
I'm all ears. Should it maybe have a single new endpoint that can take GETs (to render the initial search form) and POSTs (to render the filled-out form + any results)? I'm open to ideas.

Reply:
Is there a reason to post to `:index` and not `:create`?

Reply:
My logic, which I am struggling with, was that the POST doesn't create anything so `create` feels odd. It provides a filtered list of works and their depositors' SUNet IDs.

Reply:
Take a look at routes for collection_search and druid_search?